.github/workflows/ci.yml

```diff
@@ -67,13 +67,19 @@ jobs:
           exit-code: 1
           severity: CRITICAL
           format: table
+      - name: Define ARCHITECTURE_NAME (master)
+        if: ${{ github.ref == 'refs/heads/master' }}
+        run: echo "ARCHITECTURE_NAME=latest-${{ matrix.architecture }}" >> $GITHUB_ENV
+      - name: Define ARCHITECTURE_NAME (tags)
+        if: ${{ github.ref != 'refs/heads/master' && (contains(github.ref, 'tags') && contains(github.ref, matrix.architecture)) }}
+        run: echo "ARCHITECTURE_NAME=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
       - name: Build and publish Docker Image to GitHub Packages Registry
         uses: VaultVulp/gp-docker-action@1.1.8
         if: ${{ env.IS_DEPLOYMENT == 'true' }}
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           image-name: rails
-          image-tag: latest-${{ matrix.architecture }}
+          image-tag: ${{ env.ARCHITECTURE_NAME }}
```

Comment:
@harry-careerjsm can we change the name of this from `ARCHITECTURE_NAME` to `IMAGE_TAG`?